pkg/rbac/sharedData.go

```diff
@@ -202,7 +204,8 @@ func (shared *SharedData) getClusterScopedResources(ctx context.Context) error {
 	ds := goqu.From(schemaTable)
 	query, _, err := ds.SelectDistinct(goqu.COALESCE(goqu.L(`"data"->>'apigroup'`), "").As("apigroup"),
 		goqu.COALESCE(goqu.L(`"data"->>'kind_plural'`), "").As("kind")).
-		Where(goqu.L(`"data"->>'_hubClusterResource'='true'`), goqu.L(`"data"->>'namespace'`).IsNull()).ToSQL()
+		Where(goqu.L("???", goqu.C("data"), goqu.Literal("?"), "_hubClusterResource"),
+			goqu.L("???", goqu.C("data"), goqu.Literal("?"), "namespace").IsFalse()).ToSQL()
```

Comment:
Just to confirm, I assume `IsFalse()` covers the case of NULL?
Does this change the sample query in the comment above?

Reply:
Is there a performance benefit? 
For the namespace part `IsNull()` seemed more intuitive.I'm ok changing if it helps in any other way.